docs/first-install.md

```diff
@@ -0,0 +1,81 @@
+# First Install
+
+This is similar to [test-deployment.md](test-deployment.md), however, this document is intended for
+the first deployment onto a server with an existing, older `mod-calendar` running.
+
+## Compile the Module
+
+First, compile the module using:
+
+```sh
+mvn install
+```
+
+Note: this will require that Docker is running to complete the integration tests. If you do not have
+Docker, or wish to skip this step, add `-DskipTests` to this command.
+
+## Declare Module
+
+The module descriptor will convey the information about the module to Okapi, as described in
+`build-process.md`.
+
+```sh
+curl -w '\n' -X POST -D - \
+  -H "Content-type: application/json" \
+  -d @target/ModuleDescriptor.json \
+  http://localhost:9130/_/proxy/modules
+```
+
+This should return successfully (2xx) and echo back the newly defined module.
+
+## Build Module
+
+To build the image, use:
+
+```sh
+docker build -t docker.ci.folio.org/mod-calendar .
+```
+
+Please note that the `.` is part of the command.
+
+If your IDE does not do any compilation itself, you will likely need to run `mvn install` first.
+
+## Deploy Module
+
+Before the module can be deployed, the name of the node must be known. The following will find it
+(usually `localhost` or, on Vagrant, `10.0.2.15`):
+
+```sh
+curl -w '\n' -X GET \
+  http://localhost:9130/_/discovery/nodes
+```
+
+Once this is placed into the generated `target/DeploymentDescriptor.json`, we can deploy it onto the
+node:
```

Comment:
It could be worth being specific about this being based upon Okapi's built in deployment, which is not intended to production environments.

Reply:
Added